keepassxc-browser/content/totp-field.js

```diff
@@ -2,6 +2,7 @@
 
 const MINIMUM_SIZE = 60;
 const ignoreRegex = /^(zip|postal).*code$/i;
+const ignoredTypes = [ 'email', 'number', 'password', 'username' ];
```

Comment:
Probably don't ignore `number`, sorry for my example which included some valid types.

Reply:
True. Let's fix that.

Reply:
Looks good to me now.